pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -161,10 +187,64 @@ func (h *Helper) OtherIPs(excludeIP string) ([]string, error) {
 	return resultIPs, nil
 }
 
+func KillExistingSocat() error {
+	_, err := os.Stat(SocatPidFile)
+	if os.IsNotExist(err) {
+		return nil
+	}
+	if err != nil {
+		return err
+	}
+	pidStr, err := ioutil.ReadFile(SocatPidFile)
+	if err != nil {
+		return err
+	}
+	pid, err := strconv.Atoi(string(pidStr))
+	if err != nil {
+		return err
+	}
+	process, err := os.FindProcess(pid)
```

Comment:
i wonder if we should be more fine-grained here.  not being able to find the pid isn't a fatal error (presumably it's a stale pid file).  

I realize right now you're not treating any of these errors as fatal, but if you do change that behavior (per my other comment above) then i guess this needs to change too.